### PR TITLE
Made some UI changes 

### PR DESCRIPTION
**Pull request by kevinjcoleman**
That make it easier to complete refreshes and log acquisitions from any page. The main thing is that it looks like this now. @sponee let me know if you have any feedback!

screen shot 2017-04-18 at 3 09 41 pm

### CODE REVIEW

File: app/views/refreshes/complete.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1>Complete this refresh for <%= @refresh.state %></h1>
+<h1>Complete this refresh for <%= @refresh.state.name %></h1>
```
Comment (sponee) [inline app/views/refreshes/complete.html.erb]: @kevinjcoleman glad to see this fix!

File: app/views/refreshes/index.html.erb [hunk 2]
```diff
@@ -44,6 +45,11 @@
         <td><%= link_to state.name, state_path(state.name) %></td>
         <td><%= refresh.acquired_at.strftime("%B %d, %Y") if refresh %></td>
         <td><%= number_with_delimiter(refresh.active_voters_in_nb, :delimiter => ',') if refresh %></td>
+        <% if state.refresh_in_progress %>
```
Comment (sponee) [inline app/views/refreshes/index.html.erb]: This is super handy.

**Comment by sponee** [comment]
👍

File: app/views/refreshes/index.html.erb (hunk 2 above)
Comment (kevinjcoleman) [inline app/views/refreshes/index.html.erb]: @sponee yup! That's why I migrated the state off the refresh so I could do things like that.

**Pull request merged**